### PR TITLE
Fix login button delayed to activate bug.

### PR DESCRIPTION
**Pull request by jess0407**
Use rg-text-control in the form
https://rockabox.tpondemand.com/entity/10711

### CODE REVIEW

File: src/rb-login-form/rb-login-form.tpl.html [hunk 1]
```diff
@@ -5,26 +5,26 @@
...
                     form="loginForm"
-                    label="Email"
-                    placeholder="Email"
-                    name="email"
-                    type="email"
-                    ng-model="emailModel"
-                    is-required="true">
-                </rb-text-control>
+                    label="Email">
+                    <input type="email"
```
Comment (jonman) [inline src/rb-login-form/rb-login-form.tpl.html]: Unfortunately, the email type will need our custom validator that we have in the old `rb-text-control`.

https://github.com/rockabox/rbx_ui_components/blob/master/src/rb-text-control/custom-validation-directive.js#L4

Comment (jess0407) [inline src/rb-login-form/rb-login-form.tpl.html]: @jonman why is that?

Comment (jonman) [inline src/rb-login-form/rb-login-form.tpl.html]: The issue is similar to the `type="ur"l` default angular validator where `type="email"` thinks that `me@example` is as a valid email address.

Comment (jess0407) [inline src/rb-login-form/rb-login-form.tpl.html]: @jonman cheers noted. I think this pr can be closed though, as I will address adding custom validators on type 'url' and 'email'  for rg-text-control on a separate pr addressing tp ticket https://rockabox.tpondemand.com/entity/11309

Comment (ohskylab) [inline src/rb-login-form/rb-login-form.tpl.html]: Would we want to merge this to `master` without having custom validators in place? Any risk it'll create regressions in CM, even if just until this is picked up in the other story?

Comment (nwhite89) [inline src/rb-login-form/rb-login-form.tpl.html]: agreed, there is a PR doing this I have seen open I've made this as Do not merge and like to see the other pr merged and finished first #391

**Comment by ohskylab** [comment]
OK to merge now #391 is through?

**Comment by nwhite89** [comment]
@jess0407 can you rebase please

**Pull request merged**